### ISSUE TITLE
Grafana 9.4.0-PRE released as 8.5.16?

### ISSUE THREAD

**Issue by Espe0n** [grafana/grafana#63338]
**What happened**:

Hi,
can it be, that you accidently released Grafana 9.4.0-PRE instead of Grafana 8.5.16?
At least package.json from the released `grafana-8.5.16.tar.gz` says so

![image](https://user-images.githubusercontent.com/39615557/218756869-bdec7541-246e-4dc1-a0f0-4e0983c6e6e1.png)

Also the release contains changes which are not in the development-branch `8.5.x`

For example
https://github.com/grafana/grafana/commit/8e3d22ca7a89d8ccc6ddfc30eccdb91d7de6d1e6

https://github.com/grafana/grafana/blob/v8.5.x/packages/grafana-data/src/text/text.ts#L27

**What you expected to happen**:
`grafana-8.5.16.tar.gz` contains Grafana 8.5.16

**How to reproduce it (as minimally and precisely as possible)**:
Download `grafana-8.5.16.tar.gz` and extract it, then check `package.json` in a file-viewer of your choice.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.5.16

**Comment by Espe0n** [comment]
Apparnetly this Commit is the reason for the Release
https://github.com/grafana/grafana/commit/05bf2419528631e6dedfd34bf853f48b4bb3a552
It got the `8.5.16`-Tag, but was intended for `9.4.0`

**Comment by iomarmochtar** [comment]
Maybe this is out of topic, but seems version 8.5.16 contains a lot of changes that not exists in version 8.5.2x , one of them is rendering feature, i've tried to ask in [Grafana slack in alerting channel](https://grafana.slack.com/archives/C043CEX9MBK/p1678777010790109) also [forum](https://community.grafana.com/t/missing-image-rendering-feature-in-grafana-8-5-21/83818) until now no one can answering for that. And recently just found that alerting provisioning API is exists in 8.5.16 but not for 8.5.2x 

Is there any mistakes while releasing version 8.5.16 @tleopold ?

**Comment by tleopold** [comment]
Yes, there was an error when we were releasing 9.4.x which caused this problem but we have moved on since then so hopefully this is no longer an issue for anyone.  Good catch though ...
